SparkPost/2web2ui
# SA-1681: API request and reducer

### What Changed
 -API request uses new query params
 - Form initializes with report options from URL


### How To Test
 -  Go to report builder (hibana)
 - Enable account UI feature flag `allow_report_filters_v2`
 -  Click `Add FIlters`
 - Verify form works correctly
 - Apply filters
 - Verify that request attaches filters under query param `query_filters`
 - Click `Add Filters`
 - Verify that filters form is correct as what was applied
 - Refresh page
 - Click `Add Filters`
 - Verify that filters form is correct as what was applied.

### To Do
- [ ] Verify it works with actual API
